### PR TITLE
Net6

### PR DESCRIPTION
**Pull request by theranger**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Added support for compiling to net6.0 target

Does this close any currently open issues?
------------------------------------------
No

Any relevant logs, error output, etc?
-------------------------------------
No

Where has this been tested?
---------------------------
**Operating System:** Windows 11

**.Net Core Verion:** net6.0

Does this introduce a breaking change?
-----------------------------------------

- [ ] Yes
- [x] No

Any other comments?
-------------------
…

### CODE REVIEW

**Comment by daviddesmet** [comment]
Is the nuget.config really needed? By default, it resolves to nuget.org

**Comment by theranger** [comment]
> Is the nuget.config really needed? By default, it resolves to nuget.org

At least in my default Visual Studio setup the packages failed to load because the repository could not be resolved.

**Comment by daviddesmet** [comment]
> At least in my default Visual Studio setup the packages failed to load because the repository could not be resolved.

That's caused because of the conditionals you added to `csproj`:

```
<ItemGroup Condition="$(TargetFramework) != 'netstandard2.1' AND $(TargetFramework) != 'net6.0'">
```

Here you are telling dotnet that it shouldn't include `Microsoft.AspNetCore.App` but is required for .NET 6. It should be kept as it was.

```
<ItemGroup Condition="$(TargetFramework) == 'netstandard2.1' OR $(TargetFramework) == 'net6.0'">
    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.0" />
    <PackageReference Include="Microsoft.AspNetCore.Identity" Version="2.2.0" />
    <PackageReference Include="Microsoft.Extensions.Identity.Core" Version="6.0.3" />
    <PackageReference Include="Microsoft.Extensions.Identity.Stores" Version="6.0.3" />
</ItemGroup>
```

Here you are telling dotnet to use libraries that are for the `netstandard2.1` framework but this conditional is only for such framework and you are setting a version that is not meant for it. This block should also be kept as it was.

To summarise, the only change needed was:

```
<TargetFrameworks>netstandard2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```
